frontend/src/packages/auth/auth-api.ts

```diff
@@ -35,6 +37,22 @@ class AuthApi extends BaseHttpApi {
 
     return await response.json<UserSignUpResponseDto>();
   }
+
+  public async signIn(
+    payload: UserSignInRequestDto,
+  ): Promise<UserSignInResponseDto> {
+    const response = await this.load(
+      this.getFullEndpoint(AuthApiPath.SIGN_IN, {}),
+      {
+        method: 'POST',
+        contentType: ContentType.JSON,
+        payload: JSON.stringify(payload),
+        hasAuth: false,
+      },
+    );
+
+    return await response.json<UserSignInResponseDto>();
```

Comment:
```suggestion
    return response.json<UserSignInResponseDto>();
```

Reply:
my ticket 15 have the same problem and when i removing await i have this error
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92929504/26426719-8cdd-4d52-a284-4ba83d74d35a)

Reply:
Yeah, same happened to me, I think we just left it with **await** in the end